Makefile

```diff
@@ -38,8 +38,12 @@ check-dist:
 	python setup.py sdist
 	twine check dist/*
 
+.PHONY: start-mongo
+start-mongo:
+	docker-compose up -d
+
 .PHONY: test
-test:
+test: start-mongo
 	pytest --cov=pydantic_odm --cov-config=setup.cfg --no-cov-on-fail
 
 .PHONY: testwatch
```

Comment:
You can add a dependency for this command as you would for the `tests` command, and remove the cyclic dependency on yourself, if not difficult?

Like:
```make
.PHONY: testwatch
testwatch: up-services
```

Reply:
Thanks for reviewing, I will fix it somewhere this week!

Reply:
Fixed it 👍 I did not have encounter any more issues, I could basically copy-paste what you proposed.